pkg/types/nutanix/platform.go

```diff
@@ -195,3 +201,20 @@ func (p Platform) GetDataSourceImageFromFailureDomain(fdName, dataSourceRefName
 
 	return nil, fmt.Errorf("not found the datasource image with reference name %q in failureDomain %q", dataSourceRefName, fdName)
 }
+
+// SetPrismAPICallTimeoutDuration sets and returns the timeout duration value for prism-api calls.
+func (p *Platform) SetPrismAPICallTimeoutDuration() time.Duration {
```

Comment:
```suggestion
func (p *Platform) SetPrismAPICallTimeoutDuration() time.Duration {
        if p.PrismAPICallTimeout == nil {
                return SetPlatformDefaults() 
        } else {
                return time.Duration(p.PrismAPICallTimeout) * time.Minute
        }
} 
```
Also maybe we should consider renaming SetDefaultPrismAPICallTimeout().

Reply:
Fixed